ONSdigital/census-rm-pubsub-adapter
# Use environment config

# Motivation and Context
Sets up a way of reading config from the environment, pattern taken from https://github.com/ONSdigital/dp-import-api/tree/develop/config

# What has changed
* Add config and tests
* Use build tag to ignore integration test in unit test run

# How to test?
Run the tests, build the docker image and run in with `make up` then try posting a test message in and see it process.

# Links
https://trello.com/c/rA7COLKd/871-pubsub-adapter-read-and-check-configuration-from-environment-5
